magma/wire_container.py

```diff
@@ -281,7 +281,9 @@ def _get_conditional_drivee_info(self):
         default_drivers = []
         for ctx in self._wired_when_contexts:
             conditional_wires.append(ctx.get_conditional_wires_for_drivee(self))
-            default_drivers.append(ctx._default_drivers.pop(self, None))
+            default_drivers.append(
```

Comment:
good to put this behind an API rather than accessing `_default_drivers` directly.

If you wanted you could also just have a specialized `pop_default_driver(drivee, default)` API instead of exposing the whole dict.